### PR TITLE
add responsive to navbar + redesign unhide button for navbar with attention catcher + redesign checkbox labels

### PR DESCRIPTION
**Pull request by tmprnc**

### CODE REVIEW

File: images/htring.svg
Comment (DashDashDashDashDash) [inline images/htring.svg:1]: this seems unused

File: fa/css/all.css
Comment (DashDashDashDashDash) [inline fa/css/all.css:1]: menacing

fontawesome has its own react component library so i will not worry about the extra bandwidth for now as this should be fairly temporary
^not like moving to v2 will be any better in regards to bandwidth lol

File: styles/var.css [hunk 1]
```diff
@@ -1,22 +1,24 @@
 /* Generic colors */
 :root {
   --lnk-hvr: #00aeff;
+  --green: #CCFF66;
+  --green-rgb: 204, 255, 102;
```
Comment (DashDashDashDashDash) [inline styles/var.css:5]: could these be redundant?

**Review by DashDashDashDashDash — CHANGES_REQUESTED** [review]
just maybe remove the one seemingly unused svg otherwise it's all good - re-porting it won't bring trouble

File: styles/var.css (hunk 1 above)
Comment (tmprnc) [reply styles/var.css:5]: not quite, the rgb version is used for transparency
i.e. rgba(var(--green-rgb), [insert alpha value here])
if you used it as is it wouldn't work

File: images/htring.svg
Comment (tmprnc) [reply images/htring.svg:1]: included it just in case we would need it in future, but yes, it is unused

**Pull request merged**